lib/tasks/award_badges.rake

```diff
@@ -0,0 +1,28 @@
+namespace :award_badges do
+
+# 
+  desc "award timekeeper, energetic and ambitious badges"
+  task award_timeline_badges: :environment do
+    current_week = Week.get_previous_week_by_date(Date.today + 5.days)
```

Comment:
I'm confused - why +`5.days`?

Reply:
thanks for catching that. :D
I was testing something.